Automattic/newspack-electionkit
# feat: profile custom post type and race/district taxonomy

### All Submissions:






### Changes proposed in this Pull Request:



Adds an Election Kit Profile Custom Post type to the plugin.  Staging for future functionality.

### How to test the changes in this Pull Request:

1. Update the plugin
2. Observe ElectionKit Profiles Menu Item
3. Add a Profile, Add a District and Race Custom Taxonomy

### Other information:
